development/extensions/tutorial_testing.rst

```diff
@@ -980,3 +980,7 @@ in your code that must be fixed.
 Travis CI also provides Build Status badges. They provide you the code in markdown
 format so you can add the badge to your repository's README so visitors can see that
 the build status of your extension.
+
+    .. code-block:: txt
+
+        [![Build Status](https://travis-ci.org/phpbb/phpbb-ext-acme-demo.svg?branch=master)](https://travis-ci.org/phpbb/phpbb-ext-acme-demo)
```

Comment:
I would remove this. People should not get this from here. They should get it from Travis. Because it could change. Plus, Travis provides other forms besides markdown. So we should only encourage people to get badges directly from the source.

Reply:
I removed it, but thought an example wouldn't hurt. But if they just copy&paste it, it will break it travis changes how this works.